common/app/implicits/FaciaContentFrontendHelpers.scala

```diff
@@ -22,6 +24,19 @@ object FaciaContentFrontendHelpers {
       imageOverride.orElse(defaultTrailPicture)
     }
 
+    def mainYouTubeMediaAtom: Option[MediaAtom] =
+      for {
+       main <- faciaContent.properties.maybeContent.map(_.fields.main)
+       atoms <-  faciaContent.properties.maybeContent.flatMap(_.atoms)
+       document <- Some(Jsoup.parse(main))
```

Comment:
@mchv mention using https://github.com/xdotai/play-json-extensions to solve the play json format issue with case class with more than 22 fields